mp/src/public/tier1/ilocalize.h

```diff
@@ -319,6 +319,43 @@ class CLocalizedStringArg<float> : public CLocalizedStringArgPrintfImpl<float>
     CLocalizedStringArg( float fValue ) : CLocalizedStringArgPrintfImpl<float>( fValue, LOCCHAR("%.2f") ) {}
 };
 
+// --------------------------------------------------------------------------
+
+#ifdef _WIN32
+#define ___WIDECHAR_PRINT_FORMAT_WIDECHAR LOCCHAR("%s")
+#define ___WIDECHAR_PRINT_FORMAT_ANSICHAR LOCCHAR("%S") // Thanks, microsoft
+#elif defined(POSIX)
+#define ___WIDECHAR_PRINT_FORMAT_WIDECHAR LOCCHAR("%ls")
+#define ___WIDECHAR_PRINT_FORMAT_ANSICHAR LOCCHAR("%s")
+#else
+#warning "ILocalize needs some string macros defined!"
+#endif
+
+template < >
+class CLocalizedStringArg<locchar_t*> : public CLocalizedStringArgPrintfImpl<locchar_t*>
```

Comment:
Does this need specialization for locchar_t*? Default handler will just store pointer itself, without doing copy.